modules/apps/image-uploader/image-uploader-web/src/main/java/com/liferay/image/uploader/web/internal/util/UploadImageUtil.java

```diff
@@ -34,7 +34,7 @@ public class UploadImageUtil {
 
 	public static final String TEMP_IMAGE_FILE_NAME = "tempImageFileName";
 
-	public static final String TEMP_IMAGE_FOLDER_NAME = "java.lang.Class";
+	public static final String TEMP_IMAGE_FOLDER_NAME = "tempImageFolderName";
```

Comment:
@slnn Please use `UploadImageUtil.class.getName()`. I believe this is what the original intent, but the author got it wrong.

Reply:
Done